.orchestra/support/revng-distributable/install.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Copying root"
+rm -rf "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root"
+mkdir -p "${TMP_ROOT}${ORCHESTRA_ROOT}/revng/root"
+
+cd "$ORCHESTRA_ROOT"
+index_files=$(echo share/orchestra/glibc.idx share/orchestra/cmake.idx share/orchestra/boost.idx \
+                   share/orchestra/clang-release.idx share/orchestra/toolchain_*.idx \
+                   | sed 's/ /\n/g' | grep -v host)
+
+EXCLUDE_LIST=$(mktemp -p "${TMP_ROOT}")
+
+while IFS= read -r index_file; do
+  if [ -e "$index_file" ]; then
+    grep -vE '(((libgcc_s\.so|libc\+\+\.so|libc\+\+abi\.so)(\.\d+)*)|llvm-link|llc)' "$index_file"
+  fi
+done <<< "$index_files" | sort > "$EXCLUDE_LIST"
+
+FILE_LIST=$(mktemp -p "${TMP_ROOT}")
+
+comm -23 <(find "$ORCHESTRA_ROOT" \( -type f -or -type l \) -printf '%P\n' | sort) "$EXCLUDE_LIST" | \
+grep -Ev -e '^revng/' \
+         -e '^share/revng/qa/' \
+         -e '^lib64/libclang.*\.so' \
+         -e 'lto-dump' \
+         -e '\.mypy_cache' -e '__pycache__' | \
+grep -E -e "^bin/" \
+        -e "^include/revng/PipelineC/" \
+        -e "^lib$" \
+        -e "^lib64/python/" \
+        -e "^lib64/[^/]*\.(bc|so)$" \
+        -e "^lib64/[^/]*\.so\.[^/]*$" \
+        -e "^lib64/gcc/x86_64-pc-linux-gnu/11.2.0/[^/]*\.(o|a)$" \
+        -e "^lib64/llvm/clang-release/lib/[^/]*\.so\.[^/]*$" \
+        -e "^lib64/llvm/clang-release/bin/llc$" \
+        -e "^lib64/llvm/clang-release/bin/llvm-link$" \
+        -e "^lib64/revng/analyses/[^/]*\.so$" \
+        -e "^libexec/gcc/x86_64-pc-linux-gnu" \
+        -e "^libexec/revng/" \
+        -e "^share/orchestra/" \
+        -e "^share/revng/pipelines/" \
+        -e "^share/revng/[^/]*\.ll$" \
+        -e "^x86_64-pc-linux-gnu/" | \
```

Comment:
What criterion did you use to put together this list?
There are no comments and it's very fragile and easy to break.

Reply:
> What criterion did you use to put together this list?

Gut feelings and checking if the `test.py` passed
I can add comments, but there's no formal way for us to say "this file is needed for runtime"